docs/zkapps/advanced-snarkyjs/events.mdx

```diff
@@ -61,6 +61,63 @@ class SmartContract {
 }
 ```
 
+### Fetching Events from an Archive Node
+
+Since Mina nodes do not store historical network information and events are not kept on-chain, emitted events are only preserved in the Archive Node. Consequently, a zkApp must fetch its previously emitted events on the Mina network from an Archive Node.
```

Comment:
```suggestion
Since Mina nodes do not store historical network information and events are not kept on-chain, emitted events are preserved only in the Archive Node. Consequently, a zkApp must fetch its previously emitted events on the Mina network from an Archive Node.
```